addons/sys_logistics/fnc_logistics.sqf

```diff
@@ -743,7 +743,8 @@ switch (_operation) do {
 
             _id = _object addAction [
                 _text,
-                {[MOD(SYS_LOGISTICS),(_this select 3 select 0),[call compile (_this select 3 select 1), call compile (_this select 3 select 2)]] call ALiVE_fnc_logistics},
+                {[MOD(SYS_LOGISTICS),(_this select 3 select 0),[missionNamespace getVariable [(_this select 3 select 1),nil], 
```

Comment:
Are you sure `_input` is a variable?
According to line 735,729,723 and so on, it's not.
Container is also not a variable.
The real fix would be to edit L735,729,723 and others to actually be code and not strings.

Reply:
Eh so what do I do?